cypress/e2e/clue/branch/student_tests/table_tool_spec.js

```diff
@@ -1,10 +1,14 @@
 import TableToolTile from '../../../../support/elements/clue/TableToolTile';
+import XYPlotToolTile from '../../../../support/elements/clue/XYPlotToolTile';
 import ClueCanvas from '../../../../support/elements/clue/cCanvas';
 import Canvas from '../../../../support/elements/common/Canvas';
 
 let clueCanvas = new ClueCanvas,
-  tableToolTile = new TableToolTile;
+  tableToolTile = new TableToolTile,
+  xyplot = new XYPlotToolTile;
+
 const canvas = new Canvas;
+const queryParams = "?appMode=qa&fakeClass=5&fakeUser=student:5&qaGroup=5&unit=example";
```

Comment:
Using the `unit=example` branch as opposed to it `unit=sas` (original) - since it did not have an XY Plot button on the toolbar.